test_quality_of_service/test/test_deadline.cpp

```diff
@@ -33,7 +33,8 @@ using namespace std::chrono_literals;
 /// Test Deadline with a single publishing node and single subscriber node
 TEST_F(QosRclcppTestFixture, test_deadline) {
   int expected_number_of_events = 4;
-  const std::chrono::milliseconds deadline_duration = 1s;
+  const std::chrono::milliseconds deadline_duration =
```

Comment:
nit: an explanation why this is different for connext (observed via CI, your testing, etc)